package.json

```diff
@@ -6,6 +6,7 @@
   "type": "module",
   "dependencies": {
     "@ctrl/tinycolor": "3.5.1",
+    "@floating-ui/dom": "1.4.4",
```

Comment:
let's get a security review for this

Reply:
https://github.com/brave/reviews/issues/1400